scripts/develop.sh

```diff
@@ -62,18 +62,24 @@ CODER_DEV_SHIM="${PROJECT_ROOT}/scripts/coder-dev.sh"
 	"${CODER_DEV_SHIM}" server --address 0.0.0.0:3000 || kill -INT -$$ &
 
 	echo '== Waiting for Coder to become ready'
-	timeout 60s bash -c 'until curl -s --fail http://localhost:3000 > /dev/null 2>&1; do sleep 0.5; done'
+	timeout 60s bash -c 'until curl -s --fail http://localhost:3000/healthz > /dev/null 2>&1; do sleep 0.5; done'
```

Comment:
Maybe you can use something simpler like:

```sh 
curl --connect-timeout 1 \
    --max-time 60 \
    --retry 60 \
    --retry-delay 1 \
    --retry-max-time 60 \
    --retry-connrefused \
    'http://localhost:3000/healthz'
```

Reply:
It's a nice suggestion, but it doesn't seem to work (exits immediately).

Reply:
You can double-check with `--retry-all-errors`, but I guess that the problem is somewhere else, so feel free to ignore!

Reply:
Ah, that seems to work, weird though since the error is conn refused `¯\_(ツ)_/¯` (`Failed to connect to localhost port 3000 after 0 ms: Connection refused`)